help/destinations/catalog/crm/salesforce.md

```diff
@@ -56,15 +56,15 @@ Also ensure that the [scopes](https://help.salesforce.com/s/articleView?id=conne
 Create the custom field of type `Text Area Long` which Experience Platform will use to update the segment status within Salesforce CRM. 
 Refer to the Salesforce documentation to [create custom fields](https://help.salesforce.com/s/articleView?id=sf.adding_fields.htm&type=5) if you need additional guidance. 
 
->[!IMPORTANT]
->
+> [!IMPORTANT]
```

Comment:
let's please keep all the IMPORTANT and NOTE tags as they were, without the added space. this comment applies for several instances throughout the doc.